src/ControlSystem/UpdateControlSystem.hpp

```diff
@@ -44,6 +44,8 @@ namespace control_system {
  *
  * The algorithm to determine whether or not to update the functions of time is
  * as follows:
+ * 0. Ensure this control system is active. If it isn't, end here and don't
```

Comment:
Doxygen doesn't support zero-based lists (at least with HTML output).

Reply:
That seems silly...but oh well. Made the list start at 1 and updated the other entries.